pkg/api/v1/schema.go

```diff
@@ -920,4 +920,9 @@ type CniNetwork struct {
 	// specified, VMI namespace is assumed.
 	// In case of genie, it references the CNI plugin name.
 	NetworkName string `json:"networkName"`
+
+	// For Multus CNI select the default network and add it to the
+	// multus-cni.io/default-network annotation. Forbidden for all
+	// other CNIs.
+	Default bool `json:"default,omitempty"`
```

Comment:
If this option only pertains to multus, lets just make it only pertain to multus and not create a caveat in the documentation. 

The underlying CniNetwork struct can be broken up like this without impacting backwards compatibility.

```
type MultusNetwork struct {	
	NetworkName string `json:"networkName"`		NetworkName string `json:"networkName"`
        Default bool `json:"default,omitempty"`
}
type GenieNetwork struct {	
	NetworkName string `json:"networkName"`		NetworkName string `json:"networkName"`
}
```

Reply:
This looked more daunting the first time around when I simply grep'd for all instances of CniNetwork, I now see that this was mostly littered throughout the tests (plus I'm much more familiar with the code than I was a few months ago).